test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java

```diff
@@ -206,6 +206,7 @@ private AccountID randomNodeFrom(HapiApiSpec spec) {
 				updateStateOf(spec);
 			}
 		} catch (Throwable t) {
+			t.printStackTrace();
```

Comment:
Even if this is very helpful for local debugging, our regression does a lot of log parsing and fails on unexpected `ERROR`-type message, so this unconditional stack trace is killer. 😁